src/features/content-answers/ContentAnswersModal.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import BoxAiLogo from '../../icon/logo/BoxAiLogo';
+import ContentAnswersModalContent from './ContentAnswersModalContent';
+import ContentAnswersModalFooter from './ContentAnswersModalFooter';
+// @ts-ignore flow import
+import Modal from '../../components/modal/Modal';
+
+import messages from './messages';
+
+import './ContentAnswersModal.scss';
+
+type Props = {
+    onRequestClose: () => void;
+};
+
+const ContentAnswersModal = ({ onRequestClose }: Props) => {
+    return (
+        <Modal
+            isOpen
+            className="ContentAnswersModal"
+            data-testid="content-answers-modal"
+            onRequestClose={onRequestClose}
+            title={
+                <>
+                    <BoxAiLogo className="BoxAiLogo" data-testid="content-answers-icon-color" width={32} height={32} />
+                    <p>
+                        <span>
+                            <FormattedMessage {...messages.contentAnswersTitle} />
+                        </span>
+                    </p>
```

Comment:
are the `p` and `span` needed? can we remove these?

the `span` seems unnecessary since FormattedMessage already wraps in a `span` and the `p` tag doesn't seem semantically correct for a title

Reply:
great point!